packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,143 @@
+
+let nodeCount = $state<number>(0);
+let deploymentCount = $state<number>(0);
+let serviceCount = $state<number>(0);
+let ingressCount = $state<number>(0);
+let routeCount = $state<number>(0);
+let pvcCount = $state<number>(0);
+let configMapCount = $state<number>(0);
+let secretCount = $state<number>(0);
+
+let unsubscribers: Unsubscriber[] = [];
+
+onMount(async () => {
+  unsubscribers.push(
+    kubernetesCurrentContextNodes.subscribe(nodes => {
+      nodeCount = nodes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextDeployments.subscribe(deployments => {
+      deploymentCount = deployments.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextServices.subscribe(services => {
+      serviceCount = services.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextIngresses.subscribe(ingresses => {
+      ingressCount = ingresses.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextRoutes.subscribe(routes => {
+      routeCount = routes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextPersistentVolumeClaims.subscribe(pvcs => {
+      pvcCount = pvcs.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextConfigMaps.subscribe(configMaps => {
+      configMapCount = configMaps.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextSecrets.subscribe(secrets => {
+      secretCount = secrets.length;
+    }),
+  );
+});
+
+onDestroy(async () => {
+  unsubscribers.forEach(unsubscriber => unsubscriber());
+});
+</script>
+
+<NavPage searchEnabled={false}  title="About Kubernetes">
+  <svelte:fragment slot="additional-actions">
+    {#if !welcome}
+    <KubeApplyYamlButton />
+    {/if}
+  </svelte:fragment>
+
+  <svelte:fragment slot="bottom-additional-actions">
+    <div class="flex grow justify-end">
+      <KubernetesCurrentContextConnectionBadge />
+    </div>
+  </svelte:fragment>
+
+  <div class="flex min-w-full h-full" slot="content">
+    {#if welcome}
+        <KubernetesEmptyPage />
+    {:else}
+    <div class="flex flex-col space-y-4 min-w-full overflow-y-auto">
+        <div class="flex flex-col gap-4 p-5">
+            <span>Here you can manage and interact with Kubernetes clusters with features like connect to clusters,
+            view workloads like deployments and services.</span>
+
+        <span>Get up and running by clicking one of the menu items!</span>
+
+        <Link class="place-self-start" on:click={() => window.openExternal('https://podman-desktop.io/docs/kubernetes')}>Kubernetes documentation</Link>
+    </div>
+
+<div class="flex flex-col gap-4 bg-[var(--pd-content-card-bg)] grow p-5">
+    {#if currentContextName}
+    <div class="text-xl">Metrics</div>
+    <div class="grid grid-cols-4 gap-4">
+        <KubernetesDashboardResourceCard type='Nodes' Icon={NodeIcon} count={nodeCount} link='/kubernetes/nodes'/>
```

Comment:
suggestion (not-blocking)

it looks like we could just iterate over an array as only argument are changing